tests/stdio-test.cc

```diff
@@ -19,5 +20,9 @@ int test_stdio()
 	TEST(strcmp(buffer, "-42") == 0,
 	     "snprintf(\"%d\", -42) gave {}",
 	     std::string_view{buffer, BufferSize});
+	sprintf(buffer, "%x", 6 * 9);
+	TEST(strcmp(buffer, "36") == 0,
+	     "sprintf(\"%x\", 6 * 9) gave {}",
+	     std::string_view{buffer, BufferSize});
```

Comment:
If you're going to create a string view anyway, you can just use operator == in the comparison rather than strcmp.

Reply:
OK, I used string_view and replaced TEST with TEST_EQUAL. How's that?